Functions/Gherkin.ps1

```diff
@@ -787,3 +787,150 @@ function ConvertTo-HashTableArray {
+                    $fixtureFilename = "{0}.Steps.ps1" -f $file.BaseName
+                    $fullName = Join-Path -Path $file.Directory -ChildPath $fixtureFilename
+                    $parsed = $parser.parse($file.FullName)
+
+                    if (Test-Path -Path $fullName) {
+                        Update-TestStubs -stubFile $fullName -parsedInput $parsed
+                    }
+                    else {
+                        $stubFile = New-Item -Path $fullName -ItemType File
+                        
+                        Add-Stubs -stubFile $stubFile -parsedInput $parsed   
+                    }
+                }
+                catch {
+                    throw $_
+                }
+            }   
+        }
+        else {
+            try {
+                $fixtureFilename = "{0}.Steps.ps1" -f $item.BaseName
+                $fullName = Join-Path -Path $item.Directory -ChildPath $fixtureFilename
+                $parsed = $parser.parse($item.FullName)
+
+                if (Test-Path -Path $fullName) {
+                    Update-TestStubs -stubFile $fullName -parsedInput $parsed
+                }
+                else {
+                    $stubFile = New-Item -Path $item.Directory -Name $fixtureFilename -ItemType File
+                    
+                    Add-Stubs -stubFile $stubFile -parsedInput $parsed
+                    
+                }
+            }
+            catch {
+                throw $_
+            }
+        }
+    }
+
+    end {
+
+    }
+
+
+}
+
+function Add-Stubs {
+    param(
+        # File to write stubs into
+        [Parameter(Mandatory = $true)]
+        [string]
+        $stubFile,
+        # Parsed Gherkin Data
+        [Parameter(Mandatory = $true)]
+        [Gherkin.Ast.GherkinDocument]
+        $parsedInput
+    )
+    
+    foreach ($scenario in $parsed.feature.children) {
+
+        $string = ""
+
+        foreach ($step in $scenario.steps) {
+            $string += New-GherkinStub -step $step
+        }
+
+        Add-Content -Value $string -Path $stubFile
+    }
+}
+
+function New-GherkinStub {
+    param(
+        # The Gherkin Step to Stub
+        [Parameter(Mandatory = $true)]
+        [Gherkin.Ast.Step]
+        $step
+    )
+    
+    $outString = $step.keyword + "`""
+    
+    $description = $step.text
+    
+    $regmatch = [regex]::new("<\w*>").matches($description)
+    
+    foreach ($match in $regmatch) {
+        $value = $match.value
+        $description = $description -replace $value, "(?$value[\s\S]*)"
+    }
+                
+    $outString += $description + "`" {`r`n"
+    
+    if ($regmatch -ne $NULL) {
+        $outString += "`tparam("
+        foreach ($match in $regmatch) {
+            $paramName = $match.value.replace('<', '$').replace('>', '') + ','
+            $outString += $paramname
+        }
+        $outString = $outString.trimEnd(',') + ")`r`n"
+    }
+    
+    $outString += "`t`$true | Should Be `$false `r`n}`r`n"
```

Comment:
The default implementation should probably call `Set-TestInconclusive` or something, rather than throwing meaningless errors